pkg/helm/charts.go

```diff
@@ -39,7 +39,7 @@ type LokomotiveChart struct {
 func ChartFromAssets(location string) (*chart.Chart, error) {
 	tmpDir, err := ioutil.TempDir("", "lokoctl-chart-")
 	if err != nil {
-		return nil, errors.Wrap(err, "creating temporary dir")
+		return nil, fmt.Errorf("creating temporariy directory: %w", err)
```

Comment:
Cool, I understand. No need to add it in a seperate commit, it will be extra work.

`temporary`

Reply:
Nice catches, thanks! That was a lot of typing :smile: Fixed.